src/LaunchDarkly.ServerSdk/Internal/DataModelDependencies.cs

```diff
@@ -16,6 +17,16 @@ internal KindAndKey(DataKind kind, string key)
             Kind = kind;
             Key = key;
         }
+
+        public bool Equals(KindAndKey other)
```

Comment:
Taken from the identical `struct CacheKey`:

https://github.com/launchdarkly/dotnet-server-sdk/blob/08263f3f1cb2da5dac13075be105515ac9a35d30/src/LaunchDarkly.ServerSdk/Internal/DataStores/PersistentStoreWrapper.cs#L450